cms/djangoapps/contentstore/course_group_config.py

```diff
@@ -187,21 +188,19 @@ def _get_content_experiment_usage_info(store, course, split_tests):  # pylint: d
         return usage_info
 
     @staticmethod
-    def get_content_groups_usage_info(store, course):
+    def get_partitions_usage_info(store, course):
         """
         Get usage information for content groups.
         """
         items = store.get_items(course.id, settings={'group_access': {'$exists': True}}, include_orphans=False)
 
-        return GroupConfiguration._get_content_groups_usage_info(course, items)
+        return GroupConfiguration._get_partition_usage_info(course, items)
 
     @staticmethod
-    def _get_content_groups_usage_info(course, items):
+    def _get_partition_usage_info(course, items):
```

Comment:
Oh dear-- we now have a method called "get_partitions_usage_info" that calls "_get_partition_usage_info".

Since _get_partition_usage_info is only used in one place, can you just move its implementation into get_partitions_usage_info and delete the helper method?